### DISCUSSION TITLE
YamlConfiguration is final

### DISCUSSION THREAD

**Discussion by fankes** [charleskorn/kaml#428] [Q&A] [answered]
The fields of the configuration file are not public and are final.

How to configure Yaml? Is this project still maintained?

https://github.com/charleskorn/kaml/blob/main/src/commonMain/kotlin/com/charleskorn/kaml/YamlConfiguration.kt#L38

**Comment by charleskorn** [comment]
I'm not clear on your question sorry - you should be able to instantiate a `YamlConfiguration` instance just like any other data class, for example `YamlConfiguration(strictMode = false)`.

**Comment by fankes** [comment, marked as answer]
Sorry, is my problem, create an new `Yaml(configuration = YamlConfiguration(...))` is ok, I have been using `Yaml.default`.
